content/Coalesce.Starter.Vue.Data/GlobalUsings.cs

```diff
@@ -1,8 +1,2 @@
 global using IntelliTect.Coalesce;
-global using IntelliTect.Coalesce.DataAnnotations;
 global using Microsoft.EntityFrameworkCore;
-global using Microsoft.Extensions.Logging;
-global using System.ComponentModel.DataAnnotations;
-global using System.ComponentModel.DataAnnotations.Schema;
-global using ItemResult = IntelliTect.Coalesce.Models.ItemResult;
-global using OrderByDirections = IntelliTect.Coalesce.DataAnnotations.DefaultOrderByAttribute.OrderByDirections;
```

Comment:
These were all deliberately curated - it is expected that in any real application, most or all of these will be used.